csrc/python_frontend/multidevice_bindings.cpp

```diff
@@ -71,11 +75,24 @@ void bindDeviceMesh(py::module& nvfuser) {
       py::arg("device_id"));
 }
 
+void bindDistributedTensor(py::module& nvfuser) {
+  py::class_<DistributedTensor> distributed_tensor(
+      nvfuser, "_DistributedTensor");
```

Comment:
Docs are omitted because the class name starts with an underscore, indicating it's hidden from the user. `nvfuser.DistributedTensor`, defined in `__init__.py`, has docs though.